lib/spdy-transport/protocol/http2/parser.js

```diff
@@ -386,6 +419,28 @@ Parser.prototype.onSettingsFrame = function onSettingsFrame(header,
       settings[name] = value;
   }
 
+  if (constants.settingsIndex[constants.settings_server.SETTINGS_ENABLE_PUSH] in settings) {
```

Comment:
So, if using strings directly, this may become difficult for changing during refactoring... Are usage of "magic strings" is good idea?